src/control/server/faults.go

```diff
@@ -176,8 +177,8 @@ func FaultConfigOverlappingBdevDeviceList(curIdx, seenIdx int) *fault.Fault {
 func FaultConfigInvalidNetDevClass(curIdx int, primaryDevClass, thisDevClass int32, iface string) *fault.Fault {
 	return serverFault(
 		code.ServerConfigInvalidNetDevClass,
-		fmt.Sprintf("IO server %d specifies fabric_iface '%s' which has a different device class '%s' than the primary server's device class '%s'",
-		curIdx, iface, code.NetClassName(thisDevClass), code.NetClassName(primaryDevClass)),
+		fmt.Sprintf("IO server %d specifies fabric_iface `%s` of class `%s` that conflicts with the primary server's device class `%s`",
```

Comment:
This is fine, but for future reference the `%q` formatting directive will take care of quoting strings for you.

Reply:
fixed.